package.json

```diff
@@ -6,22 +6,26 @@
   "author": "Jawad Tahir",
   "license": "MIT",
   "devDependencies": {
+    "bigchaindb-driver": "^4.0.2",
     "browser-sync": "^2.13.0",
+    "composer-cli": "^0.19.9",
+    "composer-playground": "^0.19.9",
+    "composer-rest-server": "^0.19.9",
+    "express": "^4.16.3",
+    "generator-hyperledger-composer": "0.19.9",
     "gulp": "^3.9.1",
     "gulp-clean-css": "^2.0.10",
     "gulp-header": "^1.8.7",
     "gulp-less": "^3.1.0",
     "gulp-rename": "^1.2.2",
     "gulp-uglify": "^1.5.4",
-    "composer-cli": "^0.19.9",
-    "composer-rest-server": "^0.19.9",
-    "generator-hyperledger-composer": "0.19.9",
```

Comment:
Why have you removed these dependencies? I think they are required if you want to integrate HL stuff.

Reply:
Hi Jawad, actually (I don't know how it happen but I guess npm or smth automatically did that), after I added my dependencies with npm install package-name --save, it also sorted rest of packages alphabetically

So currently file is:
```
  "devDependencies": {
    "bigchaindb-driver": "^4.0.2",
    "browser-sync": "^2.13.0",
    "composer-cli": "^0.19.9",
    "composer-playground": "^0.19.9",
    "composer-rest-server": "^0.19.9",
    "express": "^4.16.3",
    "generator-hyperledger-composer": "0.19.9",
    "gulp": "^3.9.1",
    "gulp-clean-css": "^2.0.10",
    "gulp-header": "^1.8.7",
    "gulp-less": "^3.1.0",
    "gulp-rename": "^1.2.2",
    "gulp-uglify": "^1.5.4",
    "path": "0.12.7",
    "yo": "^2.0.2"
},
```
Composer packages still in package.json, but only in different location.